src/app/layout.tsx

```diff
@@ -18,7 +20,12 @@ export default function RootLayout({
 }>) {
   return (
     <html lang='hu'>
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <div>
```

Comment:
Ez a div felesleges itt, enélkül is jól lesz. Kivéve ha stílusozzuk majd, de akkor inkább a body-n tegyük ezt.